GeneratorInterface/Core/interface/PythiaHepMCFilterGammaGamma.h

```diff
@@ -0,0 +1,49 @@
+#ifndef PYTHIAHEPMCFILTERGAMMAGAMMA_h
+#define PYTHIAHEPMCFILTERGAMMAGAMMA_h
+
+//
+// Package:    GeneratorInterface/GenFilters
+// Class:      PythiaHepMCFilterGammaGamma
+// 
+// Original Author:  Matteo Sani
+//
+//
+
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "GeneratorInterface/Core/interface/BaseHepMCFilter.h"
+#include "FWCore/Framework/interface/EDFilter.h"
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+namespace edm {
+	  class HepMCProduct;
+}
+
+#include "TH1D.h"
```

Comment:
where is this used? In any case a filter does not seem to me the right place to fill histograms, even though this is present in a couple of them in GenFilters...
This dependency should be removed

Reply:
thanks for spotting this, this was a leftover from the original PythiaFilterGammaGamma.h in CMSSW_9_3_1. We removed this in 05c3cd62956 .